#Camera topic here
camera_topic: "/camera/color/image_raw"
depth_camera_topic: "/camera/depth/image_rect_raw"
camera_info_topic: "/camera/color/camera_info"
queue_size: 1

# Set your detection topic here
detection_topic: "/darknet_ros/bounding_boxes"

# Set your output topic here
tracker_topic: "/sort_track"
pose_topic: "/sort_track/Detection_pose"

# Threshold for Costs between detections and tracks
cost_threshold: 15

# min_hits
min_hits: 1

# max age
max_age: 5

image_view:

  enable_opencv: true
  wait_key_delay: 1
  enable_console_output: true
